addon/templates/components/frost-link.hbs

```diff
@@ -1,26 +1,26 @@
 {{#if (eq design 'info-bar')}}
-  <div class='content '>
-    <div class=' text'>
-      <div class='svg'>{{frost-icon icon=icon}}</div>
-      {{yield}}
-      </div>
-  </div>
+    <div class='content '>
+        <div class=' text'>
+            <div class='svg'>{{frost-icon icon=icon}}</div>
+          {{#if linkTitle}}{{linkTitle}}{{else}}{{yield}}{{/if}}
```

Comment:
I'd still like to see it documented in the dummy app. I had no idea this existed nor would I had I not looked at this PR.